app/styles/ember-rdfa-editor/_c-formatting-marks.scss

```
@@ -0,0 +1,43 @@
+
+/*
+* Adapted from https://github.com/guardian/prosemirror-invisibles/blob/main/src/css/invisibles.css 
+* 
+* Copyright 2018 The Guardian
+* 
+* Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated documentation files (the "Software"), to deal in the Software without restriction, including without limitation the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and to permit persons to whom the Software is furnished to do so, subject to the following conditions:
+* 
+* The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.
+* 
+* THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+* 
+*/
+.say-editor {
+
+  .invisible {
+    &::before {
+      position: relative;
+      caret-color: inherit;
+      color: gray;
+      display: inline-block;
+      font-weight: 400;
+      font-style: normal;
+      line-height: initial;
+      width: 0;
+      top: 0;
+      left: 0;
+      z-index: 1;
+    }
+    &.invisible--space::before {
+      content: '.';
```

```suggestion
      content: '\00B7';
```

https://www.compart.com/en/unicode/U+00B7

is a bit easier to distinguish from a real period